docs/release-notes.md

```diff
@@ -23,12 +36,11 @@ Starting with this release, ignition-validate binaries are signed with the
 
 ### Bug fixes
 
+- Fix failure disabling nonexistent unit with systemd ≥ 252
+- Don't relabel a mount point that already exists
 - Document that `hash` fields describe decompressed data
 - Clarify documentation of `passwordHash` fields
 - Correctly document Tang `advertisement` field as optional
-- Fix failure disabling nonexistent unit with systemd ≥ 252
-- Don't relabel a mount point that already exists
-- qemu: Use distro-specific `modprobe` path when loading kernel modules
```

Comment:
This is a cleanup that probably doesn't actually affect anyone, so on further reflection I've dropped the note.